app/assets/javascripts/application.js

```diff
@@ -12,3 +12,16 @@
 //
 //= require rails-ujs
 //= require_tree .
+function ready(fn) {
+  if (document.readyState != 'loading'){
+    fn();
+  } else {
+    document.addEventListener('DOMContentLoaded', fn);
+  }
+}
+
+ready(function() {
+  document.getElementById("user_avatar").onchange = function () {
+    document.getElementById("uploadFile").value = this.value;
```

Comment:
Yes, that is the purpose of the text box. still need to change id to follow conventions